Lombiq.Hosting.Tenants.EnvironmentRobots/Models/EnvironmentRobotsOptions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.Hosting;
+
+namespace Lombiq.Hosting.Tenants.EnvironmentRobots.Models;
+
+/// <summary>
+/// Further configuration options for the module.
+/// </summary>
+public class EnvironmentRobotsOptions
+{
+    /// <summary>
+    /// Gets or sets a value indicating whether to add a meta tag and response header with noindex, nofollow values when
+    /// the app is not running in a production environment. It adds the tags when <see cref="IsProduction"/> is <see
+    /// langword="false"/>. When set it overrides <see cref="HostEnvironmentEnvExtensions.IsProduction"/>'s result,
+    /// which the module is using by default.
+    /// </summary>
+    public bool? IsProduction { get; set; }
```

Comment:
I don't think this needs to be nullable.
At least we did not use any nullable properties in most of our `Options` classes.

Reply:
It should be nullable to, so by default it can use `IHostEnvironment.IsProduction()` (if it's null) so we don't need to configure it unless:
> in case ASP.NET Core should be told it runs in Production but not actually (like in staging, that should be as close to production as possible).

Reply:
I see thanks for clarifying. This is resolved in this case.